### ISSUE TITLE
Remove dependency on platformio for workshop

### ISSUE THREAD

**Issue by willprice** [jda0/hedgehog-clock#1]
Whilst platformio is great for library development, I think it's going to be an additional hurdle and confusion if we were to use it in the workshop.

How easy is it to extra a bunch of libs and the skeleton sketch so that we don't have to depend on platformio?

**Comment by jda0** [comment]
Should be as easy as copying contents of `/lib` to `%USERPROFILE%/Documents/Arduino/libraries`; either copying `hh_server.*` into a new folder in the same or keeping it where it is; and merging `hh_clock.*` into a `.ino` file.

Advantages of PlatformIO:
 - Cross-platform (including mac in theory, can test?)
 - Default C conventions and sensible project structure, including project-level libs (so, for example, challenge2's `drawFace` doesn't clash with a dependency in another project).
 - Can use familiar IDE if they program or we want to inspire programming - including Atom or VSCode
 - Platform setup is automatically managed, which is one of our bigger headaches. Basically, `platformio run -t upload` is magic.

Disadvantages:
 - Requires extra installation (python package/extension) and IDE

I find PlatformIO really really useful (bias: I like IntelliSense and I dislike Arduino IDE and all derivatives) so I'm not going to change the master project. Actually, I would really like VSCode to be part of the core software package we ask IT to install because making kids write Node or Python in notepad is _awful_. Anyway, feel free to create a new branch.

:v:

**Comment by willprice** [comment]
I strongly disagree, although I loathe the arduino ide I think works very well in the workshop setting. All your points above apply to a real software project but for an introduction to programming I think they serve as a distraction from the real goal. 

I appreciate this is your workshop and you're free to disregard my comments but I really think you should reconsider.

**Comment by jda0** [comment]
@willprice https://github.com/jda0/hedgehog-clock/commit/f1b873bf01c693b557d537d7788e32512e46b204
  
Leaving open until tested further with both PlatformIO and Arduino IDE.
